setup.cfg

```diff
@@ -0,0 +1,7 @@
+[isort]
+known_first_party = cabot
+known_third_party = celery,jenkins,freezegun,twilio,requests
+multi_line_output = 5
```

Comment:
I see style 0 [applied in current code](https://github.com/arachnys/cabot/blob/master/cabot/cabotapp/admin.py). It's also default, if I'm not mistaken. To be frank, I consider style 5 rather ugly. Is a change to 5 really needed?

PS: https://github.com/timothycrosley/isort#multi-line-output-modes